src/components/overview/Overview.jsx

```diff
@@ -21,8 +21,8 @@ const Overview = ({productId}) => {
     selectedProductName: '',
     selectedProductSlogan: '',
     selectedProductStyle: {},
+    selectedProductFeature: [],
```

Comment:
For this one, I actually tried to do that but I don't know if this happens because I'm using react-hook or not.

When I just pass the product detail object to the child component to use its detail property, the Test says the props is undefined such as features and category.

So, I assigned detail props such as selectedProductSlogan and selectedProductFeature at the parent component and then passed to the child component. In this way, I was able to use it for the conditional rendering.

When I tried to pass just big product object to the child component and then used the conditional rendering to use map function, I get test error as the below. If I don't do the conditional rendering, react says the map function is not averrable for undefined props.
![image](https://user-images.githubusercontent.com/65844592/148093034-36254f54-8b90-4411-a938-656a15220246.png)

![image](https://user-images.githubusercontent.com/65844592/148093052-866532e4-847c-436d-8d4d-2f006d14a670.png)

Reply:
If I do check the selectedProductDetail for conditional rendering, I get the same error that I got when I did not do the conditional rendering. 
That's why I decided to assign detail props at the parent component and pass them into the child component. 
Let me know your thought on this!

![image](https://user-images.githubusercontent.com/65844592/148100464-c3e67ef6-17e4-41e1-a4c3-8f35d8537064.png)

Reply:
I think it's definitely worth looking why this error occurs, but for now your workaround probably won't be a problem. I'll make a chore ticket about it.